controllers_v2/keysend.ctrl.go

```diff
@@ -41,13 +41,14 @@ type KeySendResult struct {
 }
 
 type KeySendResponseBody struct {
-	Amount          int64  `json:"amount"`
-	Fee             int64  `json:"fee"`
-	Description     string `json:"description,omitempty"`
-	DescriptionHash string `json:"description_hash,omitempty"`
-	Destination     string `json:"destination,omitempty"`
-	PaymentPreimage string `json:"payment_preimage,omitempty"`
-	PaymentHash     string `json:"payment_hash,omitempty"`
+	Amount          int64             `json:"amount"`
+	Fee             int64             `json:"fee"`
+	Description     string            `json:"description,omitempty"`
+	DescriptionHash string            `json:"description_hash,omitempty"`
+	Destination     string            `json:"destination,omitempty"`
+	CustomRecords   map[string]string `json:"customRecords" validate:"omitempty"`
```

Comment:
damn, just realizing we have camel case here but snake case for the other attributes :(

Reply:
yeah that sucks.

Reply:
shall we add `custom_records` also? then we can deprecate the CustomRecords in the future?

Reply:
Here we can just add the snake case as the only thing since this is a totally new field anyway. It's the request body that is more tricky.